README.md

```diff
@@ -1,24 +1,263 @@
+![eldorado-logo](https://user-images.githubusercontent.com/84629565/211169801-59cc4f73-65db-4784-81db-c591b74b87ce.png)
+
+**El Dorado Backend** is a luxary car rental application where the user can register new accounts, login and reserve a set of different cars. It is built and connected by using two different repos, including Back-end(Rails) and Front-end(React/Redux).
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Rails</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Luxary Cars List endpoint]**
+- **[Devise authentication to access Reservations]**
+- **[JWT Authorization upon login and register]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- API Docs -->
+
+## <img src="https://emojipedia-us.s3.amazonaws.com/source/microsoft-teams/337/spiral-notepad_1f5d2-fe0f.png" width="23" height="20"/> API Documentation<a name="api-docs"></a>
+
+- Here is the API documentation of the project [El Dorado API-Docs](https://eldorado.onrender.com/api-docs/)
+![api-docs](https://user-images.githubusercontent.com/84629565/212746269-ce103339-03ab-4ef6-aa5e-fea72dd7c378.png)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ER DIAGRAM-->
+## :card_index: Entity Relationship Diagram <a name="er-diagram"></a>
+![er-diagram](https://user-images.githubusercontent.com/84629565/212745874-93da1ede-2e1e-49d8-9593-e5475d56e5b6.png)
+
+<!-- LIVE DEMO -->
+## <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/539px-React-icon.svg.png" width="23" height="20"/> React Frontend <a name="react-frontend"></a>
+- Here is the Frontend part of the project [El Dorado Frontend](https://github.com/michael-duke/ElDorado-Frontend)
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link] :smiley: not yet available
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+### Prerequisites
+
+In order to run this project you need:
+`Git` and `Ruby`
+```
+ gem install rails
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install
+```
+
+Create Database (Mandatory)
+
+```
+rails db:create
+```
+
+If you have made a migration then run this command
+
+```
+rails db:migrate
+```
+```
+rails db:seed
+```
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+   rails s -p 3001 # # for serving the API on localhost:3001
+```
+
+Run tests
+```
+bundle exec rspec
+```
```

Comment:
- [ ] When I run the tests according to your instructions, this happens:

![image](https://user-images.githubusercontent.com/57421823/212815031-1289b024-68dc-4dc3-ab2f-358d0738ddf5.png)

## Explanation

This is because some tests are using rswag, and not Rspec.

## Solution

Just be more specific in your instructions:

```bash
rspec spec/models
```
Result:

![image](https://user-images.githubusercontent.com/57421823/212815307-c821a3d5-53f8-4f7c-926a-4f614274873e.png)

And for the rest of the tests:

```bash
rake rswag
```

Result:

![image](https://user-images.githubusercontent.com/57421823/212815404-8b2565fc-f4ec-4fc7-9503-f5f5fdc48415.png)

Therefore, in your readme please be more specific like this:

```bash
bundle exec rspec spec/models
rake rwag
```

That way, your user will never get a failed test again 💪

Reply:
@Meltrust thank you for your feedback, 
- Did you `bundle install` before running the command `bundle exec rspec` because for me it's working fine, I added `bundle exec`  since different users have different versions of `bundler gem`

Reply:
> @Meltrust thank you for your feedback 🚀 ,
> 
>     * Did you `bundle install` before running the command `bundle exec rspec` because for me it's working fine, I added `bundle exec`  since different users have different versions of `bundler gem`
> 
>     * Because with what you proposed the person is regenerating the swagger folder, which shouldn't happen
> 
>     * Also I'm curious what the failed description was cause it seem only the Auth tests are failing the rest is passing

Thank you for pointing out this.  These are the errors:

![image](https://user-images.githubusercontent.com/57421823/212838025-6598bf3c-c8da-4540-b620-f48c866ee37e.png)

![image](https://user-images.githubusercontent.com/57421823/212838101-7a7faa3e-36e3-44e4-8139-8c30324544ea.png)

![image](https://user-images.githubusercontent.com/57421823/212838149-bd7c3d11-e928-4a05-be0d-1be8717a8dba.png)

Used this command:

```bash
bundle exec rspec spec
```

Another idea if you don't want your users to regenerate the swagger folder, would be to just use:

```bash
bundle exec rspec spec/models
```

That way, no folder will be regenerated, and there won't be any non-passing tests on screen.

Reply:
I see what the issue is, I believe you don't have the
# Devise secret key
DEVISE_JWT_SECRET_KEY=5f33f59d2733e3a982718060645336936c4d3446a16903e281387c6e1d53550f56747c2ff3855c13dea5a93c6502a32d85ad6d06c216e7e9761e5776de04f967

Reply:
This is the beauty of PR comments and excellent communication in a professional environment.  If I get the chance to do the next review, I will issue a commend in the reviewer's dashboard. 🥇